### PR TITLE
Fix "Test Metric "Notebook CPU Usage" (ODS-178) when cpu usage is null

### PR DESCRIPTION
**Pull request by rkamble5**
Signed-off-by: rkamble <rkamble@redhat.com>

### CODE REVIEW

**Comment by rkamble5** [comment]
Tested Successfully here :- https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/909/

**Review by jgarciao — APPROVED** [review]
lgtm

File: tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot [hunk 1]
```diff
@@ -100,8 +100,12 @@ Read Current CPU Usage
     ${Expression} =    Set Variable
     ...    sum(rate(container_cpu_usage_seconds_total{prometheus_replica="prometheus-k8s-0", container="",pod=~"jupyterhub-nb.*",namespace="rhods-notebooks"}[1h]))
     ${resp} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${Expression}
-    [Return]    ${resp.json()["data"]["result"][0]["value"][-1]}
-
+    IF    ${resp.json()["data"]["result"]} == []
+        ${list_of_data}    Set Variable    0
```
[the remark this answers is not in the document]
Comment (jgarciao) [reply tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot:104]: Yes, ${data} or ${cpu_usage} would be better. Could you change it, @rkamble5?

[the remark this answers is not in the document]
Comment (rkamble5) [reply tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot]: Done

**Review by jgarciao — APPROVED** [review]
lgtm

**Review by tarukumar — APPROVED** [review]
LGTM

**Pull request merged**